config/build.js

```diff
@@ -0,0 +1,25 @@
+const path = require( 'path' );
+const pkg = require( '../package.json' );
+
+const cwd = process.cwd();
+
+const entryPoints = pkg.entries.map( entry => (
+    path.resolve( cwd, entry )
+) );
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+module.exports = {
+    bundle: true,
+    entryPoints,
+    format: 'iife',
+    minify: isProduction,
+    outdir: path.resolve( cwd, './public/js/build' ),
+    sourcemap: isProduction ? false : 'inline',
+    target: [
+        'chrome89',
+        'edge89',
+        'firefox90',
+        'safari13',
+    ],
+};
```

Comment:
Since CI has been upgraded and the browser targets are now the same, this config has been moved out to a place that both test and production builds can share without deviation.